File: Data Structures/Arrays/Cpp/Read_an_array.cpp
// Write a Program to Read an array.

#include<iostream>
using namespace std;

int main(){
    int Array[5];
    for(int i = 0; i < 5; i++){
        cin >> Array[i];
    }
    for(int i = 0; i < 5; i++){
        cout << Array[i] << " ";
    }
    return 0;
}

File: Data Structures/Arrays/Java/Read_an_array.java
// Write a Program to Read an array.

import java.util.Scanner;

public class Read_an_array{
    public static void main(String[] args){
    
        Scanner read = new Scanner(System.in);
        int Array[] = new int[5];
        System.out.println("Enter the elements of the array: ");
        for(int i = 0; i < 5; i++){
            Array[i] = read.nextInt();
        }
        for(int i = 0; i < 5; i++){
            System.out.print(Array[i] + " ");
        }
        read.close();
    }
}